client-app-example/ifc-cli.config.js

```diff
@@ -14,10 +14,10 @@ module.exports = function(frameRouter) {
     },
     {
       locale: 'en-US',
-      hostRootUrl: window.location.origin,
+      hostRootUrl: window.location.origin + '/#/',
```

Comment:
The host app now needs to pass the hash part of the URL down to the client app, since it's an always-present part of the host routing, and a key part of generating host URLs